libbeat/beat/info.go

```diff
@@ -42,6 +43,8 @@ type Info struct {
 	Monitoring struct {
 		DefaultUsername string // The default username to be used to connect to Elasticsearch Monitoring
 	}
+	LogConsumer consumer.Logs //otel log consumer
```

Comment:
Will this eventually hold metrics and trace consumer?

Reply:
If necessary, yes it would go here.  But Beat metrics look more like Logs, so the current plan is to send logs and metrics as `Logs`.